docs/ops.html

```diff
@@ -1545,6 +1545,51 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
         <td>kafka.server:type=BrokerTopicMetrics,name=BytesRejectedPerSec,topic=([-.\w]+)</td>
         <td>Rejected byte rate per topic, due to the record batch size being greater than max.message.bytes configuration. Omitting 'topic=(...)' will yield the all-topic rate.</td>
       </tr>
+      <tr>
```

Comment:
I would suggest adding a section in Monitoring called "Tiered Storage Monitoring" (similar to KRaft Monitoring Metrics). It would help the reader easily identify what metrics are relevant to what feature.

Reply:
Done